src/Favicon/Favicon.php

```diff
@@ -185,7 +185,10 @@ private function getFavicon($url, $checkDefault = true) {
 
         // See if it's specified in a link tag in domain url.
         if (!$favicon) {
-            $favicon = $this->getInPage($url);
+            $favicon = trim($this->getInPage($url));
+        }
+        if (substr($favicon, 0, 2) === '//') {
+            $favicon = 'https:' . $favicon;
```

Comment:
Yes indeed. But in practice, all the examples I see are for sites proposing HTTP and HTTPS at the same time.